packages/help-center/src/components/back-button.tsx

```diff
@@ -36,21 +52,8 @@ export const BackButton = ( { onClick, backToRoot = false, className }: BackButt
 
 	return (
 		<Button className={ buttonClassName } onClick={ onClick || defaultOnClick }>
-			<Icon icon={ chevronLeft } size={ 18 } />
-			{ __( 'Back', __i18n_text_domain__ ) }
+			<Icon icon={ key === 'default' ? <DragIcon /> : chevronLeft } size={ 18 } />
+			{ buttonText ?? defaultButtonText }
```

Comment:
Rather than making the entire header a back button would it be better functionally if we only make the icon clickable? I think it comes across a little confusing from a UI perspective. Doing this would also fix the click and drag issue that is happening since the drag is also a button. What are your thoughts?

Reply:
Here is a video of what I mean.

https://github.com/user-attachments/assets/ec2ed3f6-2d5b-489a-925f-73c598398dfd

Reply:
Makes sense to me as well 👍